labs/cordentity.md

```diff
@@ -1,8 +1,3 @@
----
-layout: default
-title: Cordentity
-parent: Archived Labs
----
```

Comment:
I believe these changes need to exist in order for https://labs.hyperledger.org/ to show appropriately. Change line 4 to `Labs`, but leave the other lines as is.

Reply:
If there are external demands to unarchive, can you add any indication of who they are and how they will participate.